Lib/bdb.py

```diff
@@ -59,6 +239,18 @@ def canonic(self, filename):
             self.fncache[filename] = canonic
         return canonic
 
+    def start_trace(self, trace_dispatch):
+        if self.backend == 'monitoring':
```

Comment:
```suggestion
        if self.monitoring_tracer:
```
Avoids the magic strings constant